executor/testdata/runner/mock_host.py

```
@@ -123,15 +125,17 @@ async def storage_read(
 		assert self.storage is not None
 		return self.storage.read(Address(account), slot, index, le)
 
+	async def remaining_fuel_as_gen(self) -> int:
+		return 2**32
+
 	async def storage_write(
 		self,
-		account: bytes,
 		slot: bytes,
 		index: int,
 		got: collections.abc.Buffer,
```

Flake8 (7.2.0)

[error] 'collections' may be undefined, or defined from star imports: base_host

(F405)